### ISSUE TITLE
Add collapsable sidemenu

### ISSUE THREAD

**Issue by kryptag** [Praqma/web-codeall#7]
the side menu is very important for the navigation of the site so this is a must have

**Comment by kryptag** [comment]
Right now the sidemenu is removed on mobile view, which is not the right way. you can find it in `wireframe.sass` where we `@extend .hidden-xs`

**Comment by lakruzz** [comment]
Here's something that looks useful

https://bootsnipp.com/snippets/featured/fancy-sidebar-navigation

**Comment by anetacamo** [comment]
I made something similar for my website and I could easily apply some styles for the menu into the bootstrap template! 
So it have nice transition and change into x when clicked. In the screenshot there is not the "X" just because I did not want to spend too much time on it. 
I moved the menu items to the left so it could be easily visible

here is with the animation "X"
https://codepen.io/anon/pen/WEKbQo

<img width="701" alt="screen shot 2017-08-25 at 14 20 21" src="https://user-images.githubusercontent.com/30343653/29713991-6131ccf0-89a1-11e7-9de3-09c2106c20b2.png">
<img width="708" alt="screen shot 2017-08-25 at 14 20 38" src="https://user-images.githubusercontent.com/30343653/29714009-6a5d8c6a-89a1-11e7-9cdc-43a34e9f8b83.png">
